build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -813,16 +824,32 @@ CONFIG_LWIP_DHCPS_MAX_STATION_NUM=8
 # CONFIG_LWIP_DHCPS_ADDITIONAL_DNS is not set
 # CONFIG_NETUTILS_XMLRPC is not set
 # CONFIG_NETUTILS_NTPCLIENT is not set
-# CONFIG_NETUTILS_WEBSERVER is not set
+CONFIG_NETUTILS_WEBSERVER=y
+CONFIG_NETUTILS_WEBSERVER_MAX_CLIENT_HANDLER=1
+# CONFIG_NETUTILS_WEBSERVER_LOGD is not set
+# CONFIG_NETUTILS_WEBSERVER_LOGE is not set
+CONFIG_NETUTILS_WEBSERVER_MAX_CLIENT_RCV_TIMEOUT=360
+CONFIG_NETUTILS_WEBSERVER_MIN_CLIENT_RCV_TIMEOUT=50
 # CONFIG_NETUTILS_FTPC is not set
 # CONFIG_NETUTILS_MDNS is not set
 # CONFIG_NETUTILS_FTPD is not set
 # CONFIG_NETUTILS_LIBCOAP is not set
 # CONFIG_NETUTILS_TFTPC is not set
 # CONFIG_NETUTILS_TELNETD is not set
 # CONFIG_NETUTILS_SMTP is not set
+CONFIG_TRT_JSON=y
```

Comment:
Is this config present in public?

Reply:
No , I don't think so.
Let me update it